Cohen-Koen.md

```diff
@@ -0,0 +1,32 @@
+﻿---
+name: "@Cohen-Koen"
+project: "Screens for Cougar MFD"
+---
+
+#  Screens for Cougar MFD
+
+##  Summary
+
+I am going to build and integrate screens with the Thrustmaster Cougar MFD, this will allow for a realistic look on MFD, I wish to make an affordable option for anyone who works with flight simulators like me. There are alternatives however I wish to make them well integrated and something that people can use as a guide for their own projects
+
+##  Plan
+
+Once my parts arrive, I will build a mount for my MFD with the screen.
+Second, I will setup a software or etc. to allow the user sim of choice to operate with the screen.
+Throughout this project I will show how this is done with good photos and documentation on Gitbook and I will share the process to help the flight simulator community
+Third, I will integrate the MFD with my Arch Linux installation so I can see a terminal on both of them as mini terminals, this will all be open source and free. Once uploaded wherever find fit, I will share my socials so people can communicate with me if they need help. 
+If you wish to email me,
+GMAIL: Cohen.Dickey@gmail.com
+I2P: D8Z@mail.i2p
+
+##  Budget
```

Comment:
Are there any other components involved in this project? If so, it'd be great if you could add it to your budget—just specify the cost as $0 if you already own the thing.

Reply:
Oh my, thanks I forgot to add the MFD themselved as screen are usually seen as a mod, I already own the Cougar MFD so I am not sure to add them to budget or not, what do you think?